top_section_styles.css

```diff
@@ -0,0 +1,254 @@
+@import url(fonts.css);
+
+:root {
+    --primary: #3077C6;
+    --primary-dark: #10367C;
+    --secondary-yellow: #F5BD41;
+    --secondary-dark-mode1: #121212;
+    --secondary-dark-mode2: #1D1E1F;
+    --secondary-dark-mode3: #4D4D4D;
+    --primary-stroke: #F3F3F4;
+    --primary-text: #383838;
+    --primary-white: #FFFFFF;
+    --secondary-text: #BFBFBF;
+    --primary-bg: #EAF0F9;
+}
+
+body {
+    margin: 0;
+}
+.big-container {
+    width: 1440px;
+    height: 900px;
+    min-height: 100vh;
+    background-image: url("images/booking_images/kastelmeccano.jpg");
+    background-repeat: no-repeat;
+    background-position: top;
+
+}
+
+.big-upper {
+    display: flex;
+    justify-content: space-between;
+    margin: 50px 100px 100px 0;
+    align-items: center;
+}
+
+.upper {
+    display: flex;
+    justify-content: space-between;
+    width: 345px;
+    align-items: center;
+    margin-top: 50px;
+}
+
+.words {
+    display: flex;
+    justify-content: space-between;
+    /*margin-top: 50px;*/
+    width: 200px;
+    align-items: center;
+}
+
+.icons {
+    display: flex;
+    justify-content: space-between;
+    /*margin-top: 50px;*/
+    width: 95px;
+    align-items: center;
+}
+
+.image, .svg {
+    width: 205px;
+    height: 40px;
+    fill: var(--primary);
+    margin-top: 50px;
+    margin-left: 100px;
+    align-items: center;
+}
+
+.stays {
+    width: 60px;
+    height: 30px;
+    color: var(--primary-white);
+    font: 24px Roboto, sans-serif;
+    font-weight: 400;
+    line-height: 28px;
+    align-items: center;
+}
+
+.attractions {
+    width: 120px;
+    height: 30px;
+    color: var(--primary-white);
+    font: 24px Roboto, sans-serif;
+    font-weight: 400;
+    line-height: 28px;
+    align-items: center;
+}
+
+.icon_night, .svg {
+    width: 30px;
+    height: 30px;
+    fill: var(--primary-white);
+    align-items: center;
+}
+
+.icon_account, .svg {
+    width: 40px;
+    height: 40px;
+    fill: var(--primary-white);
+    align-items: center;
+}
+
+
+.hr {
+    margin-top: -104px;
+    margin-left: 1000px;
+    width: 50px;
+    height: 4px;
+    background-color: var(--secondary-yellow);
+    border: none;
+}
```

Comment:
@IharDorinel I think it's an animation to see that the element has this underline is active - therefore it's better to set display none and change it depends on hover of attraction or stays

Reply:
also set hr into attractions/stays near its text content and set positions relative to attractions/stays class and absolute to hr (also with bottom value less than 0)

Reply:
@romos21 I wanna make it in such way - so is it possible to set pseudoelement's width depending on word's length?
word:hover:after {
    content: "";
    display: block;
    width: 50px;
    height: 4px;
    background: #F5BD41;
    margin: auto;
    position: relative;
}